jmxtrans.sh

```diff
@@ -17,7 +17,7 @@ fi
 
 JAVA_HOME=${JAVA_HOME:-"/usr"}
 LOG_DIR=${LOG_DIR:-"."}
-LOG_FILE=${LOG_FILE:-"/dev/null"}
+STDOUT_REDIRECT_FILE=${STDOUT_REDIRECT_FILE:-"/dev/null"}
```

Comment:
Naming might not be the best ever, but it's not a hack, it's a good practice to capture stdout on a daemon process. You can think about it as 2 different levels of logging.